class/metallb.yml

```diff
@@ -9,3 +15,19 @@ parameters:
           - metallb/component/main.jsonnet
         input_type: jsonnet
         output_path: metallb/
+      - output_path: metallb/10_metallb_helmchart
+        input_type: helm
+        output_type: yaml
+        input_paths:
+          - metallb/helmcharts/metallb
+        helm_values:
+          speaker:
+            secretName: ${metallb:speaker:secretname}
+          configInline:
+            address-pools:
```

Comment:
We already have a use case where we need to configure multiple address pools. I would simply expose the address-pools array in parameters and have `address-pools: ${metallb:address_pools}` here.

Reply:
See #2 . There might be more advanced configs in the future. Therefore make it over writable on a config level might make sense in the future.